projects/frontend/data-pipelines/gui/projects/data-pipelines/src/lib/components/data-job/pages/executions/data-job-executions-page/data-job-executions-page.component.ts

```diff
@@ -61,7 +61,8 @@ export class DataJobExecutionsPageComponent
         OnTaurusModelInit,
         OnTaurusModelLoad,
         OnTaurusModelChange,
-        OnTaurusModelError
+        OnTaurusModelError,
+        OnInit
```

Comment:
nit: newly added OnInit with no new usages?

Reply:
It was complaining that ngOnInit should implement it for some reason. I figured it wouldn't hurt.